MIGRATION.md

```diff
@@ -1,5 +1,9 @@
 # Migration guides
 
+## 3.7.1 to 3.7.2
+
+- `Radar.getTripOptions()` and `Radar.getTrackingOptions` return typed `RadarTripOptions` and `RadarTrackingOptions` respectively rather than nesting within an `options` key.
```

Comment:
@lmeier Can you add a migration code snippet (example below)

Reply:
https://github.com/radarlabs/capacitor-radar/pull/86/commits/ff5525e6e1a21c8cfbd28e39d6b35e45ad6a830c